specification/azsadmin/resource-manager/fabric/readme.md

```diff
@@ -0,0 +1,92 @@
+# Commerce Admin
+    
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Commerce Admin.
+
+---
+## Getting Started 
+To build the SDK for Commerce Admin, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+### Basic Information 
+These are the global settings for the Commerce API.
+
+``` yaml
+title: FabricAdminClient
+description: Fabric Admin Client
+openapi-type: arm
+tag: package-2016-05-01
+```
+
+### Tag: package-2016-05-01
+
+These settings apply only when `--tag=package-2016-05-01` is specified on the command line.
+
+``` yaml $(tag) == 'package-2016-05-01'
+input-file:
+    - "Microsoft.Fabric.Admin/2016-05-01/Fabric.json"
+    - "Microsoft.Fabric.Admin/2016-05-01/EdgeGateway.json"
+    - "Microsoft.Fabric.Admin/2016-05-01/EdgeGatewayPool.json"
+    - "Microsoft.Fabric.Admin/2016-05-01/FabricLocation.json"
+    - "Microsoft.Fabric.Admin/2016-05-01/FileShare.json"
+    - "Microsoft.Fabric.Admin/2016-05-01/InfraRole.json"
+    - "Microsoft.Fabric.Admin/2016-05-01/InfraRoleInstance.json"
+    - "Microsoft.Fabric.Admin/2016-05-01/IpPool.json"
+    - "Microsoft.Fabric.Admin/2016-05-01/LogicalNetwork.json"
+    - "Microsoft.Fabric.Admin/2016-05-01/StoragePool.json"
+    - "Microsoft.Fabric.Admin/2016-05-01/StorageSystem.json"
+    - "Microsoft.Fabric.Admin/2016-05-01/Volume.json"
+    - "Microsoft.Fabric.Admin/2016-05-01/LogicalSubnet.json"
+    - "Microsoft.Fabric.Admin/2016-05-01/MacAddressPool.json"
+    - "Microsoft.Fabric.Admin/2016-05-01/Operations.json"
```

Comment:
The name of the file is oncorrect. operations.json vs Operations.json. Please fix